src/ziggurat/init.clj

```diff
@@ -32,11 +37,11 @@
 
 (defn- start-messaging-consumers [args]
   (start-messaging-connection args)
-  (messaging-consumer/start-subscribers (get args :stream-routes) (ziggurat-config)))
+  (messaging-consumer/start-subscribers (get args :stream-routes) (get args :batch-routes) (ziggurat-config)))
```

Comment:
Can these be clubbed? `(get args :stream-routes) (get args :batch-routes)` to something like `(get args :event-routes)`.

Reply:
We cannot club this because start subscriber has different logic for `:stream-routes` and `:batch-routes`.

Reply:
deconstruct args